packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1511,7 +1513,9 @@ abstract class RenderBox extends RenderObject {
   ///
   /// This function should never return a negative or infinite value.
   ///
-  /// See also examples in the definition of [computeMinIntrinsicWidth].
+  /// See also:
+  ///
+  ///  * Examples in the documentation for [computeMinIntrinsicWidth].
```

Comment:
The thing to look at usually comes first: [foo], which is the bar.

Reply:
Fixed, and below.